src/Altinn.Apps/KubernetesWrapper/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM microsoft/dotnet:sdk AS build-env
+WORKDIR /app
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
```

Comment:
Why copy only `*.csproj` files in this step, when you copy all files in the second step after this?

Reply:
Suggestion: only have one `COPY . ./` and have `RUN dotnet restore` after that one.

Reply:
I just copied the microsoft example, I guess it has something to do with caching the different layers. (https://docs.docker.com/engine/examples/dotnetcore/). Is it considered better to do it all on one step?

Reply:
Don't know if it's considered better. Just that `COPY . ./` also copies the `*.csproj` file. Not sure if it skips the file since it's already in the docker image.

Keep it :)